lib/puppetserver/ca/local_certificate_authority.rb

```diff
@@ -76,7 +101,7 @@ def format_time(time)
         time.strftime('%Y-%m-%dT%H:%M:%S%Z')
       end
 
-      def create_master_cert(ca_key, ca_cert)
+      def create_master_cert(ca_key=@key, ca_cert=@cert)
```

Comment:
Do we need these args here? We don't seem to be actually passing anything to this ever, and even in the test, I think calling `subject.key` is the same as just letting it use `@key`.

Reply:
I guess I really just don't like the look of the fields as default args. I kind of think with this refactor, we are establishing a model where this CA class is an entity that should only be able to do things with its verified, saved internals. So I'm in favor of removing them.